source/common/http/conn_manager_impl.cc

```diff
@@ -91,6 +93,9 @@ ConnectionManagerImpl::~ConnectionManagerImpl() {
     if (codec_->protocol() == Protocol::Http2) {
       stats_.named_.downstream_cx_http2_active_.dec();
     } else {
+      if (isWebSocketConnection()) {
+        stats_.named_.downstream_cx_websocket_active_.dec();
+      }
       stats_.named_.downstream_cx_http1_active_.dec();
```

Comment:
Yes, i thought about it and my rational was that WebSocket is strictly a subset of a HTTP/1.1 connection, and as such, http1 counters should be reflected as well. That said, I like your point about total connections to a machine.